terraform/modules/kubernetes/azure_redis.tf

```diff
@@ -70,11 +82,13 @@ resource "azurerm_redis_cache" "redis-queue" {
   sku_name                      = var.redis_queue_sku_name
   minimum_tls_version           = var.redis_minimum_tls_version
   public_network_access_enabled = var.redis_public_network_access_enabled
+  zones                         = var.redis_replica_zones
```

Comment:
Do we want the default of this to be `["1","2"]` meaning all the environments will have the zone configuration like this?

Reply:
This we agreed to be default for a minimum 2 availability zones this is only valid for production we are not going to roll out the rest to other environments